### DISCUSSION TITLE
User Permissions

### DISCUSSION THREAD

**Discussion by Marcos8060** [mosesmbadi/easymed#4] [Q&A]
Apart from the patients, which category of user does not have access to the patient referrals modal?

**Comment by mosesmbadi** [comment]
Approach it this way, when a patient walks in, their details are captured by the receptionist. The receptionist sends the data to _triage_, where a nurse proceeds. So three people can now see that data including the doctor. After the _triage_, the patient goes to consultation. In _consultation_, the doctor sends the patient to _phlebotomy_ for a test. The phlebotomists sees the data (which up to this point includes the **patient's demographic**, **contacts**, **next of kin** and **insurance information** as well as the **test request**). The phlebotomist takes the sample and sends it to the labtech (or to the Lab generally). Up to this point, the receptionist, the nurse, lab tech and phlebotomist can see this data. The next stage is where strict permissions are enforced. After the test is done and results generated **ONLY** the doctor who sent the **test request** is allowed to see the **TEST RESULTS**. Other information can be seen by the rest fine, but **THE TEST RESULTS** and reports generated afterward can **ONLY** be seen by the doctor who sent the test request. The patient however does not see the **TEST RESULTS** he/she will only see the report the doctor generates/writes.
